### PR TITLE
[Doc]Update databricks doc for 22.12[skip ci]

### LINKED ISSUE #6982: [DOC] Update documentation to indicate support for DPP on Databricks
## Report incorrect documentation

Now that issue #3143 is closed, we should update the getting started guide on Databricks to indicate that DPP can be used.

**Location of incorrect documentation**
https://github.com/NVIDIA/spark-rapids/blob/branch-22.12/docs/get-started/getting-started-databricks.md#limitations

**Describe the problems or issues found in the documentation**
The first step indicates to disable DPP which is now unnecessary on Databricks

---

## Report needed documentation

**Describe the documentation you'd like**
The first indicating disable DPP can now be removed.

### PR DESCRIPTION
**Pull request by viadea**
Update databricks doc for 22.12:
1. Remove DPP=off config
2. Add link to https://github.com/NVIDIA/spark-rapids-container 

fixes #6982

### CODE REVIEW

**Comment by nvliyuan** [comment]
verified, approved

**Review by nvliyuan — DISMISSED** [review]

File: docs/FAQ.md [hunk 1]
```diff
@@ -264,9 +264,6 @@ efficient to stay on the CPU instead of going back and forth.
 
 Yes, DPP still works.  It might not be as efficient as it could be, and we are working to improve it.
```
Comment (sameerz) [inline docs/FAQ.md:265]: @jlowe is this still a valid statement?  Can DPP be more efficient?

Comment (jlowe) [reply docs/FAQ.md:265]: The second sentence can be removed, we're no longer running the subquery twice as we once were.

Comment (sameerz) [inline docs/FAQ.md:265]:
```suggestion
Yes, DPP works.  
```

**Review by sameerz — CHANGES_REQUESTED** [review]

**Comment by tgravescs** [comment]
this should be associated with https://github.com/NVIDIA/spark-rapids/issues/6982 and we need to remove the spark.databricks.delta.optimizeWrite.enabled= false option as well

**Comment by viadea** [comment]
> this should be associated with #6982 and we need to remove the spark.databricks.delta.optimizeWrite.enabled= false option as well

Removed. want to review again? @tgravescs @sameerz

**Review by sameerz — APPROVED** [review]

File: docs/get-started/getting-started-databricks.md [hunk 2]
```diff
@@ -156,6 +143,14 @@ cluster.
 7. Once you’ve added the Spark config, click “Confirm and Restart”.
 8. Once the cluster comes back up, it is now enabled for GPU-accelerated Spark.
 
+## RAPIDS Accelerator for Apache Spark Docker container for Databricks
+
+Github repo [spark-rapids-container](https://github.com/NVIDIA/spark-rapids-container) provides the 
+Dockerfile and scripts to build a custom Docker containers with RAPIDS Accelerator for Apache Spark.
```
Comment (jlowe) [inline docs/get-started/getting-started-databricks.md:149]:
```suggestion
Dockerfile and scripts to build custom Docker containers with RAPIDS Accelerator for Apache Spark.
```

**Review by jlowe — COMMENTED** [review]
Minor nit, otherwise lgtm.

**Review by tgravescs — DISMISSED** [review]

**Review by jlowe — APPROVED** [review]

**Review by sameerz — APPROVED** [review]

**Comment by sameerz** [comment]
build

**Pull request merged**